tests/test_commands.py

```
@@ -176,6 +176,94 @@ def test_hunt_new_increments_id(self, runner, temp_workspace):
+
+        assert result.exit_code == 0, result.output
+        match = re.search(r"Created (H-\d+)", result.output)
+        assert match, f"Could not find hunt ID in output: {result.output}"
+        hunt_id = match.group(1)
+
+        hunt_files = list((temp_workspace / "hunts").rglob(f"{hunt_id}.md"))
+        assert len(hunt_files) == 1
+        content = hunt_files[0].read_text()
+
+        assert "credential-access" in content, (
+            "Expected auto-derived tactic 'credential-access' in hunt frontmatter; " "got hunt content:\n" + content
```

Ruff (0.15.12)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)